src/lib/Theme.ts

```diff
@@ -177,7 +179,7 @@ export default class Theme {
         const companyListOptions = {};
         if (!companyList.items.length) {
             throw new CommandError(
-                ErrorCodes.NO_COMPANY_FOUND.message,
+                ErrorCodes.NO_COMPANY_FOUND.message(company_type, getBaseURL().replace('api', 'partners')),
```

Comment:
can it be moved into `src/lib/CommandError.ts` @manishprajapati52

Reply:
Added common function to get projects domain.